pandas/tools/util.py

```diff
@@ -74,9 +94,33 @@ def to_numeric(arg, errors='raise'):
     >>> import pandas as pd
     >>> s = pd.Series(['1.0', '2', -3])
     >>> pd.to_numeric(s)
+    0    1.0
+    1    2.0
+    2   -3.0
+    dtype: float64
+    >>> pd.to_numeric(s, downcast='float')
+    0    1.0
+    1    2.0
+    2   -3.0
+    dtype: float16
```

Comment:
I know but it doesn't matter if numpy supports it
NO algos support it not is it tested in pandas 
I don't want to have an easy way to convert things to a pretty much unsupported dtype